app/utils/discogs.schemas.ts

```diff
@@ -1,8 +1,10 @@
 import z from "zod";
 
 const AlbumSchema =  z.object({
+  cover_image:z.string(),
+  title: z.string().optional(),
   year: z.string().optional(),
```

Comment:
same with year?

```suggestion
  year: z.string(),
```

Reply:
year is not defined for every object. Some objects are missing the year key value pair. I left this as optional and inserted a ternary operator in Result to handle the undefined case.